Src/AuxFunctions.c

```diff
@@ -756,54 +752,6 @@ void changeRunMode(){
 
 }
 
-void forceRunMode(uint8_t wantedMode){
```

Comment:
Are you sure this function is to be removed?

Reply:
Yes, because the forceRunMode() is used in the production testing DUT FW and the changeRunMode() is the one used in the user FW